### PR TITLE
Usage Instructions improvements

### PR DESCRIPTION
**Pull request by brunobowden**
- 'ios' as suggested folder name rather than 'xcode'
- ‘suggested’ rather than ‘preferred’ for folder structure
- more ascii art for folder structure
- add j2objcConfig example
- improve link for list of tasks that can be enabled or disabled
- formatting and indentation for code sections
- reference to project structure

### CODE REVIEW

**Comment by brunobowden** [comment]
@advayDev1, @confile - please take a look, lots of minor improvements

**Comment by advayDev1** [comment]
LG, like the ios name better.  i think your PR and commit descriptions are out-of-date?

**Comment by confile** [comment]
I will have a look in the morning.

**Comment by brunobowden** [comment]
@advayDev1 - the PR description doesn't automatically get updated but I do update the commit message. I've manually copied it across. Anything look out of place?

**Comment by advayDev1** [comment]
your commit says:
```
- 'ios' as suggested folder name rather than 'xcode'
- ‘suggested’ rather than ‘preferred’ for folder structure
- more ascii art for folder structure
- add j2objcConfig example
- improve link for list of tasks that can be enabled or disabled
- formatting and indentation for code sections
- reference to project structure
```

i don't think lines 2, 3, 4 are relevant to this commit.
i think you are reusing branches and/or rebasing to an old branch so combining an old commit message?

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ It may start as an empty project and allows you to gradually shift over code fro
 Android application. See the section below on [Folder Structure](#folder-structure).
 
 **Note: the `plugins { id 'com.github.j2objccontrib.j2objcgradle' }` syntax does
-not work for the j2objc plugin. You must use the old buildscript style.
+not yet work for the j2objc plugin. You must use the old buildscript style.
```
Comment (confile) [inline README.md]: You must use the old buildscript syntax listed below.

File: README.md [hunk 2]
```diff
@@ -63,12 +63,12 @@ The suggested folder structure is:
     │   │   ├───binaries        // Contains test binary: testJ2objcExecutable/debug/testJ2objc
     │   │   └───j2objcOutputs   // j2objc generated headers and libraries
     │   ├───lib                 // if library jar has source, then j2objc can translate it
-    │   └───src                 // src/main/java/... i.e. main code for translation
-    └───xcode
+    │   └───src                 // src/main/java/... shared code for translation
```
Comment (confile) [inline README.md]: I think the shared project should include at least one dependent project. This is what you have in most cases. You don't have only one shared project. When shared includes dependent project then you need a settings.gradle and then the challenge begins. You can only use one settings.gradle in your project structure and Android Studio needs one as well. 
We should definitely show where to put this file in the project structure. As for me this is still unclear.

File: README.md [hunk 3]
```diff
@@ -79,12 +79,14 @@ You can disable tasks performed by the plugin using the following configuration
 Of the tasks that can be disabled, the major tasks (like ``j2objcTest``), are listed in
 [J2objcPlugin.groovy#L58](https://github.com/j2objc-contrib/j2objc-gradle/blob/master/src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcPlugin.groovy#L58).
 
-### Interim development
+
+### Development
```
Comment (confile) [inline README.md]: Plugin Development

File: README.md (hunk 2 above)
Comment (brunobowden) [inline README.md]: I'll do this in a separate commit

File: README.md (hunk 3 above)
Comment (brunobowden) [inline README.md]: changed

File: README.md (hunk 1 above)
Comment (brunobowden) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (confile) [inline README.md]: Okay I am fine with that!

**Pull request merged**